actioncable/app/javascript/action_cable/index.js

```diff
@@ -1,58 +1,45 @@
+import adapters from "./adapters"
 import Connection from "./connection"
 import ConnectionMonitor from "./connection_monitor"
 import Consumer from "./consumer"
 import INTERNAL from "./internal"
+import * as logger from "./logger"
 import Subscription from "./subscription"
 import Subscriptions from "./subscriptions"
```

Comment:
BTW, I think we can export most of these directly here instead of importing and exporting down below. e.g.
```js
export Subscription from "./subscription"
export Subscriptions from "./subscriptions"
```
Curious if that reduces the compile file size at all.

Reply:
Seems that the syntax has to be:
```js
export { default as Subscription } from "./subscription"
export { default as Subscriptions } from "./subscriptions"
```
since they are default exports from the internal modules rather than named exports. I made the change, but it doesn't reduce the file size (they're equivalent from rollup's perspective).

Reply:
Updated: https://github.com/rails/rails/blob/2e8fb53720ce8c12ed183f683330c7b16c288433/actioncable/app/javascript/action_cable/index.js